receiver/awsxrayreceiver/receiver.go

```diff
@@ -117,6 +117,7 @@ func (x *xrayReceiver) Shutdown(_ context.Context) error {
 func (x *xrayReceiver) start() {
 	incomingSegments := x.poller.SegmentsChan()
 	for seg := range incomingSegments {
+		seg.Ctx = x.obsrecv.StartTraceDataReceiveOp(seg.Ctx)
```

Comment:
`ctx := x.obsrecv.StartTraceDataReceiveOp(seg.Ctx)`

Every operation starts from `seg.Ctx` not from previous returned.

Reply:
Changed in the latest commit